README.md

```diff
@@ -50,6 +71,14 @@ The **foundry-app-name** determines the URL for the Redfish service.
 
 The cloud foundry makes use of the following files: requirements.txt, runtime.txt, and Profile. So they should exists in the same directory as emulator.py.
 
+### Docker
+
+This command runs the container with the built-in mockup:
+
+```bash
+docker run --rm dmtf/redfish-interface-emulator:latest
+```
+
```

Comment:
How are configuration files handle when running with Docker.  Say one needs to edit emulator-config.json (to enable HTTPS) and then add server.cert and server.key to the root directory?  Simple as making the changes, before the invocation?

Reply:
You can pass things in as arguments like it were the standalone tool. However, if you need to pass in files, you would need to expose the path to the container.

For example,, this can be done with the mockup server:
`docker run --rm -v <path-to-mockup>:/mockup dmtf/redfish-mockup-server:latest -D /mockup`

`-v <path-to-mockup>:/mockup` will mount local path `<path-to-mockup>` in the container at `/mockup`.

So, for the emulator, you'll need to use `-v` like above to share a folder with the container, and use whatever arguments you have for the emulator at the end to point to files in the shared folder.